specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-02-09/Quotas.json

```diff
@@ -0,0 +1,313 @@
+                    "application/json"
+                ],
+                "produces": [
+                    "application/json"
+                ],
+                "parameters": [
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+                    },
+                    {
+                        "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+                    },
+                    {
+                        "$ref": "Quotas.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Quotas.json#/definitions/QuotaList"
+                        }
+                    },
+                    "404": {
+                        "description": "NOT FOUND"
+                    }
+                },
+                "x-ms-pageable": {
+                    "nextLinkName": null
+                }
+            }
+        }
+    },
+    "definitions": {
+        "Quota": {
+            "description": "Holds quota information used to control resource allocation.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Quota properties.",
+                    "x-ms-client-flatten": true,
+                    "$ref": "#/definitions/QuotaProperties"
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "../2015-12-01-preview/Compute.json#/definitions/Resource"
+                }
+            ]
+        },
+        "QuotaProperties": {
+            "description": "Properties for a Quota",
+            "type": "object",
+            "properties": {
+                "availabilitySetCount": {
+                    "description": "Maximum number of availability sets allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "coresLimit": {
+                    "description": "Maximum number of core allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "virtualMachineCount": {
+                    "description": "Maximum number of virtual machines allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "vmScaleSetCount": {
+                    "description": "Maximum number of scale sets allowed.",
+                    "format": "int32",
+                    "type": "integer",
+                    "minimum": 0
+                },
+                "maxAllocationStandardManagedDisksAndSnapshots": {
```

Comment:
This is a 45 character long property name.  This gets converted into a C# property and a Cmdlet parameter.

Could you come up with a much shorter property name?

Perhaps you could have two quota values?  standardManagedDisksLimit and standardSnapshotLimit?

Reply:
This is the exact string that is used in UX. I believe this is also how the azure team uses this.
Unless this is blocking, splitting this into two parameters is a long effort.

Reply:
This is just Azure Stack. Azure does not use our quota type. Has this already shipped? If yes, changing this requires a bit of data migration and coordination with UX, PS, etc. Mess. Try not to use "max" in any quota property names going forward. If not shipped yet - please get rid of the "max" prefix and stick with the "count" suffix convention.